src/routes/articles/update-article.e2e.test.js

```diff
@@ -0,0 +1,199 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const { fixtures } = require('../../../test/utils')
+
+describe('PATCH /articles/:id', () => {
+	let user
+	let accessToken
+	let article
+	let updatedInfo 
+	before(async ()=> {
+		user = await fixtures.insertUser() 
+		const body = { id: user.id, email: user.email }
+		accessToken = fixtures.generateAccessToken(
+			{user : body}
+		)
+		article = await fixtures.insertPost({
+			userId : user.id,
+			type : 'article'
+		})
+		updatedInfo = {
+			title : faker.random.word(),
+			image : faker.image.imageUrl(),
+			content : faker.random.words(),
+			published : faker.datatype.boolean(),
+			articleId : article.id,
+			createdOn : article.createdAt.toISOString(),
+			userId : article.userId
```

Comment:
if this is being added to this because we need the fields for the expectation, we can just refer to the article fields when it gets to that.

that is, we can have

```js
expect(res.body).to.eql(
						{
							status: 'success',
							data: {
							  message: 'Article successfully updated',
							  title: updatedInfo.title,
							  image: updatedInfo.image,
							  article: updatedInfo.content,
							  published: updatedInfo.published,
							  articleId : article.id,
							  createdOn : article.createdOn,
							  userId : article.userId
							}
```

let's not add fields that are not part of the actual update

Reply:
i gave an example of what to do in the block of code in the earlier comment

Reply:
check their data types and delete them right?

Reply:
no

Reply:
```js
expect(res.body).to.eql(
						{
							status: 'success',
							data: {
							  message: 'Article successfully updated',
							  title: updatedInfo.title,
							  image: updatedInfo.image,
							  article: updatedInfo.content,
							  published: updatedInfo.published,
							  articleId : article.id,
							  createdOn : article.createdOn,
							  userId : article.userId
							}
```

Reply:
i remember now